src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -237,15 +238,24 @@ protected void parseSplitSampleData(Variant variant, String fileId, String study
             // Samples may remove the trailing fields (only GT is mandatory),
             // so the loop iterates to sampleFields.length, not formatFields.length
             for (int j = 0; j < sampleFields.length; j++) {
+                if (formatFields[j].equals("GT")) {
```

Comment:
You could use the `formatField` variable to access the field in a more homogeneous way inside a single branch.